README.md

```diff
@@ -6,10 +6,18 @@
 - **Prize Draw**: Create a prize draw and reward the winners with airdrops or offline prizes.
 - **Data**: Download a CSV file containing the wallet addresses of your followers, followings, and/or anyone who engages with your post.
 
-
 ## Running Locally
 
-First, run the development server:
+1. Copy/Paste the .env.dist file and complete with your own values.
+
+3. Run the following commands:
+```bash
+npm install
+# or
+yarn install
+```
+
+4. Run the development server:
```

Comment:
A few things:

1. I think the first instruction should be to clone the repo
2. The numbering is not correct. You are missing the number 2.
3. The command for starting the server is repeated. You can remove number 3.

Reply:
Oh yes, thank's for points 1 and 2.
But for point 3, the command is not repeated since point 3 is install with npm (or yarn) and point 4 starts the server, or I am missing something ?

Reply:
```suggestion
1. Clone the repo:
```bash
git clone https://github.com/jelilat/lensdrop.git
```

2. Copy/Paste the .env.dist file and complete it with your own values.

3. Run the development server:
```bash
npm install
# or
yarn install
```